rectools/models/ease.py

```diff
@@ -54,7 +56,12 @@ class EASEModel(ModelBase[EASEModelConfig]):
     verbose : int, default 0
         Degree of verbose output. If 0, no output will be provided.
     num_threads: int, default 1
-        Number of threads used for `recommend` method.
+        Number of threads used for recommendation ranking on cpu.
+    recommend_use_gpu_ranking: Optional[bool], default ``None``
+        Flag to use gpu for recommendation ranking. If ``None``, `implicit.gpu.HAS_CUDA` will be
+        checked before inference.
+        This attribute can be changed manually before calling model `recommend` method if you
```

Comment:
Don't mix up method arguments and class attributes. They are not necessarily named the same. You should specify here which exactly attribute to change.

Reply:
fixed only for EASE before we approve final words